packages/components/src/hooks/useSnackbar.tsx

```diff
@@ -0,0 +1,45 @@
+import { useContext } from 'react'
+
+import * as Toast from 'react-native-toast-notifications'
+
+import {
+  SNACKBAR_DEFAULT_OFFSET,
+  modifyToastOptions,
+} from '../components/Snackbar/Snackbar'
+import { SnackbarContext } from '../components/Snackbar/SnackbarProvider'
+
+/**
+ * Passthrough of useToast hook which has been modified to allow for optional
+ * offset setting
```

Comment:
Think we may want to update this as this is doc that will appear on hover for consumers and they shouldn't need to know or care that we're using a particular library under the hood or how we're modifying it.

Perhaps something like:
```suggestion
 * Hook to manipulate the Snackbar with `show`, `hide`, or `isOpen` to query if already present
 *
 * Must be wrapped in `SnackbarProvider` to function
```
?